swApiClient.cpp

```diff
@@ -1,10 +1,27 @@
-#include "swApiClient.h"
+#include <stdlib.h>
+#include <string.h>
 #include <iostream>
+#include "swApiClient.h"
 
+using namespace std;
+using json = nlohmann::json;
 
-/* static member init */
-std::string swApiClient::BASE_URL = "http://api.shapeways.com";
-std::string swApiClient::API_VERSION = "v1";
+/* Helper Functions */
+struct curl_slist * swApiClient::SetAuthHeader() {
+  char *headerStr = (char*)malloc(250);
+  strcpy(headerStr, "Authorization: Bearer ");
+  strcat(headerStr, this->accessToken.c_str());
+  cout << headerStr << endl;
```

Comment:
funny mix of C and C++ styles. I'm fine with it.

Reply:
It's 2018 Hans, I'm platform-fluid.